webapp/components/FileUrlSelector.js

```diff
@@ -31,24 +31,28 @@ export default class FileUrlSelector extends React.Component {
 
     return (
       <div>
-        <h2>What file do you wish to load?</h2>
-        <div className="aligned-form-elements-wrapper">
+        <Typography variant="h6" className="input-label">What file do you wish to load?</Typography>
+        <Box display="flex" alignItems="flex-end" justifyContent="space-between" className="input-with-button">
           <TextField
             id="nwb-url-input"
             placeholder="Paste a URL pointing to an NWB v2 file"
             // helperText="Insert a public url or local absolute path of an NWB file"
-            className = 'aligned-form-element input'
-            variant="outlined"
+            className = 'input-form-control'
+            placeholder="Paste an URL pointing to an NWB v2 file"
+            margin="0"
+            InputLabelProps={
+              { shrink: true }
+            }
             onChange={ evt => this.updateInputValue(evt) }
           />
           <Button
+            color="primary"
             id="load-file-button"
-            variant="outlined"
+            variant="contained"
             onClick={this.handleClickLoadFile}
-            className = 'aligned-form-element button'
             disabled={this.state.inputValue.length <= 5}
-          >Load NWB file</Button>
-        </div>
+          >LOAD NWB FILE</Button>
```

Comment:
Text should be lower case, the text-transform is about styling (should be the default for all buttons)

Reply:
This on has uppercase text in figma, so did the same